src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -114,81 +23,131 @@ def read_exomiser_json_result(exomiser_result_path: Path) -> dict:
+        """Return gene symbol from Exomiser result entry."""
+        return result_entry["geneSymbol"]
+
+    @staticmethod
+    def find_gene_identifier(result_entry: dict) -> str:
+        """Return ensembl gene identifier from Exomiser result entry."""
+        return result_entry["geneIdentifier"]["geneId"]
+
+    def find_relevant_score(self, result_entry: dict):
+        """Return score from Exomiser result entry."""
+        return round(result_entry[self.ranking_method], 4)
+
+    def extract_pheval_gene_requirements(self) -> [PhEvalGeneResult]:
+        """Extract data required to produce PhEval gene output."""
         simplified_exomiser_result = []
-        for result in self.exomiser_json_result:
-            if self.ranking_method in result:
-                simplified_exomiser_result = SimplifiedExomiserGeneResult(
-                    result, simplified_exomiser_result, self.ranking_method
-                ).create_simplified_gene_result()
+        for result_entry in self.exomiser_json_result:
+            if self.ranking_method in result_entry:
+                simplified_exomiser_result.append(
+                    PhEvalGeneResult(
+                        gene_symbol=self.find_gene_symbol(result_entry),
+                        gene_identifier=self.find_gene_identifier(result_entry),
+                        score=self.find_relevant_score(result_entry),
+                    )
+                )
+
         return simplified_exomiser_result
 
-    def simplify_variant_result(self) -> [dict]:
-        """Simplify Exomiser json output into variant results."""
+
+class PhEvalVariantResultFromExomiserJsonCreator:
+    def __init__(self, exomiser_json_result: [dict], ranking_method: str):
+        self.exomiser_json_result = exomiser_json_result
+        self.ranking_method = ranking_method
+
+    @staticmethod
+    def find_chromosome(result_entry: dict) -> str:
+        """Return chromosome from Exomiser result entry."""
+        return result_entry["contigName"]
+
+    @staticmethod
+    def find_start_pos(result_entry: dict) -> int:
+        """Return start position from Exomiser result entry."""
+        return result_entry["start"]
+
+    @staticmethod
+    def find_end_pos(result_entry: dict) -> int:
+        """Return end position from Exomiser result entry."""
+        return result_entry["end"]
+
+    @staticmethod
+    def find_ref(result_entry: dict) -> str:
+        """Return reference allele from Exomiser result entry."""
+        return result_entry["ref"]
+
+    @staticmethod
+    def find_alt(result_entry: dict) -> str:
+        """Return alternate allele from Exomiser result entry."""
+        return result_entry["alt"]
+
+    def find_relevant_score(self, result_entry) -> float:
+        """Return score from Exomiser result entry."""
+        return round(result_entry[self.ranking_method], 4)
+
+    def extract_pheval_variant_requirements(self) -> [PhEvalVariantResult]:
```

Comment:
```python
@staticmethod
def extract_pheval_variant_requirements(self, exomiser_json_result: [dict], ranking_method: str) -> [PhEvalVariantResult]:
```
Also consider the name? Although this isn't really a public API it's  important so you understand its purpose next month :)